dietpi-software.html

```diff
@@ -2525,6 +2525,38 @@ <h3>Domoticz</h3>
 						</div>
 					</div>
 				</div>
+				<!-- Start details for portfolio project 206 openHAB-->
+				<div id="openhab" class="row single-project">
+					<div class="col-md-4"><a href="https://dietpi.com/docs/assets/images/dietpi-software-homeautomation-openhab.jpg" target="_blank" rel="noopener"><img src="images/dietpi-software/openhab-screenshot-small.jpg" alt="openHAB Screenshot" loading="lazy"></a></div>
```

Comment:
The large versions of the screenshots are still missing in both cases, right? Put them into `images/dietpi-software/` dir and URL path as well.

Reply:
no, the large version is the external link to the docs image. But maybe I'm not understanding completely what you mean

Reply:
Ah sorry, I missed that they are pointing to the docs. All fine then.